content/pt-br/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -0,0 +1,356 @@
+e salve como gmsa-crd.yaml.
+A seguir, instale o CRD com `kubectl apply -f gmsa-crd.yaml`
+
+### Instale webhooks para validar usuários GMSA
+
+Dois webhooks precisam ser configurados no cluster Kubernetes para popular e validar 
+as referências de especificação de credenciais GMSA no nível do Pod ou contêiner:
+
+1. Um webhook de mutação que expanda as referências para as GMSAs,
+(por nome de uma especificação de Pod) dentro de toda a especificação de credencial, 
+no formato JSON dentro da especificação do Pod.
+
+1. Um webhook de validação garante que todas as referências para GMSAs estão 
+autorizadas a serem usadas pela conta de serviço do Pod.
+
+A instalação dos webhooks acima e dos objetos associados requer as etapas abaixo: 
+
+1. Crie um par de chaves de certificado (que será usado para permitir que o 
+contêiner do webhook se comunique com o cluster)
+
+1. Instale a `Secret` com o certificado acima.
+
+1. Crie um Deployment para a lógica principal do webhook.
+
+1. Crie as configurações de webhook de validação e de mutação, referentes ao Deployment.
+
+Um [script](https://github.com/kubernetes-sigs/windows-gmsa/blob/master/admission-webhook/deploy/deploy-gmsa-webhook.sh) 
+pode ser usado para implantar e configurar os webhooks GMSA e objetos associados 
+mencionados acima. O script pode ser executado com a opção ```--dry-run=server``` 
+para possibilitar que você possa revisar as alterações antes que sejam aplicadas 
+no seu cluster.
+
+O [template YAML](https://github.com/kubernetes-sigs/windows-gmsa/blob/master/admission-webhook/deploy/gmsa-webhook.yml.tpl) 
+usado pelo script também pode ser usado para implantar os webhooks e objetos 
+associados manualmente (com as substituições apropriadas para os parâmetros).
+
+<!-- steps -->
+
+## Configurar GMSAs e nós Windows em Active Directory
+
+Antes que os Pods no Kubernetes possam ser configurados para usar GMSAs, as GMSAs apropriadas precisam ser provisionadas no Active Directory como descrito na 
+[documentação do Windows GMSA](https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/getting-started-with-group-managed-service-accounts#BKMK_Step1). 
+Nós `worker` Windows (que são parte do cluster Kubernetes) precisam ser configurados no 
+Active Directory para acessar as credenciais secretas associadas com a GMSA apropriada, 
+como descrito na [documentação Windows GMSA](https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/getting-started-with-group-managed-service-accounts#to-add-member-hosts-using-the-set-adserviceaccount-cmdlet).
+
+## Crie recursos de especificação de GMSA
+
+Com o `GMSACredentialSpec CRD` instalado (como descrito anteriormente), 
+recursos customizados contendo recursos de especificação de credenciais GMSA podem 
+ser configurados. A especificação de credencial GMSA não contém segredos nem dados 
+sensíveis. É informação que o contêiner runtime pode usar para descrever a apropriada 
```

Comment:
"agente de execução de contêiner" would be better suited instead of "contêiner runtime".